can_testbench.py

```diff
@@ -340,18 +377,248 @@ def updatePlot(self):
     def closeEvent(self, event):
         # Perform any cleanup or save data here
         permitClose = True
-        for sig in self.msg.signals:
-            if sig.graph:
-                permitClose = False
-
+        
         if permitClose:
```

Comment:
Why keep permitClose if it is now always true?

Reply:
At the time I was considering if I might want to add some sort of condition, I will remove it.